src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scripts/CompositorWindow.cs

```diff
@@ -155,13 +155,34 @@ private void CompositeGUI()
                                 .ToArray());
 
                             compositionManager.CaptureDevice = supportedDevices[selectedIndex];
+
+                            if (supportedDevices[selectedIndex] == FrameProviderDeviceType.AzureKinect)
+                            {
+                                EditorGUILayout.EndHorizontal();
+                                EditorGUILayout.Space();
+                                EditorGUILayout.BeginHorizontal();
+
+                                GUIContent occlusionLabel = new GUIContent("Occlusion Mode", "The occlusion mode used for compositing.");
```

Comment:
>The occlusion mode used for compositing [](start = 94, length = 39)

I would maybe word this as "The occlusion mode used to determine if real-world or holographic content is displayed using depth information from supported cameras."